app/containers/LoggedInPage.js

```diff
@@ -1,12 +1,25 @@
 import { connect } from 'react-redux';
+import { push } from 'react-router-redux';
+import { bindActionCreators } from 'redux';
 import LoggedIn from '../components/LoggedIn';
+import userActions from '../actions/user';
 
 const mapStateToProps = (state) => {
   return state;
 };
 
-const mapDispatchToProps = (dispatch) => { // eslint-disable-line no-unused-vars
-  return {};
+const mapDispatchToProps = (dispatch) => {
+  const user = bindActionCreators(userActions, dispatch);
+  return {
+    onLogout: (data) => {
+      user.logout(data);
+      dispatch(push('/'));
+    }
+  };
 };
 
 export default connect(mapStateToProps, mapDispatchToProps)(LoggedIn);
+
+
+
+
```

Comment:
Here too

Reply:
Good spot. Btw I've been using `prettier` for a while now and we never had to bother about code style ever again. May make sense to add `yarn format` to re-format all the code, that also includes trailing newlines etc..

Reply:
Yeah definitely agreed! I use `prettier` in all my projects now. I think it's definitely worthwhile to add.